lambdas/services/batch_update_ods_code.py

```diff
@@ -196,31 +206,5 @@ def build_progress_dict(self, dynamodb_records: list[dict]) -> dict:
                         "[multiple ods codes in records]"
                     )
 
-        self.logger.info(f"Totally {len(progress_dict)} patients found in record.")
+        logger.info(f"Totally {len(progress_dict)} patients found in record.")
         return progress_dict
-
-
-def setup_logging_for_local_script():
-    importlib.reload(logging)
-
-    logging.basicConfig(
-        level=logging.INFO,
-        format="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s",
-        datefmt="%d/%b/%Y %H:%M:%S",
-        stream=sys.stdout,
-    )
-
-
-if __name__ == "__main__":
-    import argparse
-
-    setup_logging_for_local_script()
-
-    parser = argparse.ArgumentParser(
-        prog="batch_update_ods_code.py",
-        description="A utility script to update the ODS Codes for all patients in a dynamoDB doc reference table",
-    )
-    parser.add_argument("table_name", type=str, help="The name of dynamodb table")
-    args = parser.parse_args()
-
-    BatchUpdate(table_name=args.table_name).main()
```

Comment:
Why have we removed this main? And have we checked for usages of this script elsewhere if it used in the actions anywhere, it may break the process if we rely on the `batch_update.main()` it will work within a python environment, but if this script is called from anywhere else it will break. Is that the intended change?